editor/find_in_files.cpp

```diff
@@ -183,7 +183,7 @@ void FindInFiles::_iterate() {
 			_current_dir = _current_dir.plus_file(folder_name);
 
 			PoolStringArray sub_dirs;
-			_scan_dir(_root_prefix + _current_dir, sub_dirs);
+			_scan_dir("res://" + _current_dir, sub_dirs);
```

Comment:
`_root_prefix` is now unused, but still defined in the `.h` file (I wonder why `-Werror` didn't catch that :/).

Reply:
Fixed.